spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRequiredDistributionAndOrdering.java

```diff
@@ -298,4 +300,33 @@ public void testRangeDistributionWithQuotedColumnNames() throws NoSuchTableExcep
         ImmutableList.of(row(7L)),
         sql("SELECT count(*) FROM %s", tableName));
   }
+
+  @Test
```

Comment:
wanted to add a `parameterized` test. But the parent class (SparkCatalogTestBase) itself is `parameterized`. So, it cannot be nested again for specific testcase I guess. 

hence added as two separate test case.

Reply:
I think the bigger problem here is that we do not have a guarantee that this file was actually sorted according to the spec. We'll need to check distributions configuration and make sure the Spark writer is actually writing sorted data.

Reply:
Can you please elaborate on how to do that? I am not very familiar with that.

I was thinking that just mentioning the corresponding table sort order id in the manifest info is enough for time travel queries. 
Also, I am wondering how the file scan task currently working without the sort order id (in case of sort order evolution)